services-js/commissions-search/src/App.js

```diff
@@ -53,12 +50,12 @@ class App extends React.Component {
   };
 
   findCheckedBoxes4GA = event => {
-    var chk_arr = document.getElementsByClassName('cb');
-    var chklength = chk_arr.length;
-    for (var k = 0; k < chklength; k++) {
-      var checkit = chk_arr[k].firstChild.checked ? true : false;
-      if (checkit === true) {
-        var children = chk_arr[k].childNodes;
+    const chkArr = document.getElementsByClassName('cb');
```

Comment:
I know we talked about commenting this code (to explain why you’re not using state) and using something lest fragile than `firstChild` and `children[1]`. 

Is there a reason you didn't go that route?

Reply:
I was waiting for Travis to finish and then going to make comments in GitHub and address the questions / comments there.

Reply:
See comments above for updates to this.